Parser/GenericParser/readLinesWithRegex.m

```diff
@@ -0,0 +1,80 @@
+function [lines] = readLinesWithRegex(fid, re_signature, stopfirst)
+%function [lines] = readLinessWithRegex(fid, re_signature, stopfirst);
+%
+% Read an in memory file line by line using
+% a regex signature until the match fails or until the end of file.
+%
+% Inputs:
+%
+% fid - integer - the file id
+% re_signature - string - a regular expr to detect a header line
+% stopfirst - boolean - switch to stop at first unmatch.
+%
+% Outputs:
+%
+% lines - a cell containing the lines
+%
+% Example:
+%
+% fid = fopen('data/testfiles/JFE/v000/20160112_0419_ACLW-USB_0341_041745_A.csv');
```

Comment:
It would be great to have a link to the s3 bucket containing this file. Would help me, and others, to see how it works properly

Reply:
you can obtain all the files by running get_testfiles.py - this is explained in the wiki https://github.com/aodn/imos-toolbox/wiki/UnitTests